tests/test_indexes.py

```diff
@@ -1029,13 +1029,23 @@ def test_timedelta_div() -> None:
     delta = dt.timedelta(1)
 
     check(assert_type(index / delta, "pd.Index[float]"), pd.Index, float)
-    check(assert_type(index / 1, "pd.TimedeltaIndex"), pd.TimedeltaIndex, pd.Timedelta)
+    check(assert_type(index / [delta], "pd.Index[float]"), pd.Index, float)
+    check(assert_type(index / 1, pd.TimedeltaIndex), pd.TimedeltaIndex, pd.Timedelta)
+    check(assert_type(index / [1], pd.TimedeltaIndex), pd.TimedeltaIndex, pd.Timedelta)
     check(assert_type(index // delta, "pd.Index[int]"), pd.Index, np.longlong)
-    check(assert_type(index // 1, "pd.TimedeltaIndex"), pd.TimedeltaIndex, pd.Timedelta)
+    check(assert_type(index // [delta], "pd.Index[int]"), pd.Index, int)
+    check(assert_type(index // 1, pd.TimedeltaIndex), pd.TimedeltaIndex, pd.Timedelta)
+    check(assert_type(index // [1], pd.TimedeltaIndex), pd.TimedeltaIndex, pd.Timedelta)
 
     check(assert_type(delta / index, "pd.Index[float]"), pd.Index, float)
+    # ZeroDivisionError
+    # check(assert_type([delta] / index, "pd.Index[float]"), pd.Index, float)
```

Comment:
I think if you did `index = pd.Index([pd.Timedelta(days=1)], dtype="timedelta64[s]")`, then we wouldn't get the div by zero.

Reply:
Thank you, that made it work!